Document DV Setup Process with Sedge (#22)
ObolNetwork/sedge: docs/docs/commands/importKey.mdx

      --custom-config string         file path or url to use as custom network config.
      --custom-deploy-block string   custom network deploy block.
      --custom-genesis string        file path or url to use as custom network genesis.

>       --distributed boolean           Import keys generated by Charon Distributed Key Generation (DKG) process

Reviewer: ```suggestion
      --distributed boolean           Import keys generated in a Distributed Key Generation (DKG) process
```
Planning ahead for an eventual Nethermind client :)
